modules/advanced-seo/advanced-seo.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * Class containing utility static methods that other advanced seo features are relying on.
+ */
+class Advanced_SEO {
+	/**
+	 * Site option name used to store front page meta description.
+	 */
+	const FRONT_PAGE_META_OPTION = 'advanced_seo_front_page_description';
+
+	/**
+	 * Old version of option name that was previously used under Free plan.
+	 */
+	const GRANDFATHERED_META_OPTION = 'seo_meta_description';
+
+	/**
+	 * Used to check whether advanced seo features are enabled.
+	 *
+	 * @param int $blog_id Optional. Defaults to current blog id if not given.
+	 *
+	 * @return bool True if advanced seo features are enabled, false otherwise.
+	 */
+	public static function is_enabled_advanced_seo( $blog_id = 0 ) {
+		if ( empty( $blog_id ) ) {
+			$blog_id = get_current_blog_id();
+		}
+
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
I think it might be better to check if a function exists then check if IS_WPCOM. 
`if( function_exists( 'has_blog_sticker' ) )`